code/modules/admin/create_object.dm

```diff
@@ -9,20 +10,14 @@
 
 	user << browse(replacetext(create_object_html, "/* ref src */", "\ref[src]"), "window=create_object;size=425x475")
 
+/datum/admins/proc/quick_create_object(mob/user)

```

Comment:
minor thing, but while (mob/user) works because of the implied var/, it's almost always better to explicitly define (var/mob/user)